page.codeberg.libre_menu_editor.LibreMenuEditor.yml

```diff
@@ -0,0 +1,42 @@
+app-id: page.codeberg.libre_menu_editor.LibreMenuEditor
```

Comment:
Can you please verify that `libre_menu_editor.codeberg.page` is actually the website codeberg allows for your username?

Reply:
My username on codeberg.org is "libre-menu-editor" and not "libre_menu_editor" and the url that I can use on codeberg (if I enable the codeberg.page feature) is "https://libre-menu-editor.codeberg.page" and not "https://libre_menu_editor.codeberg.page".

Since hfiguiere told me that "page.codeberg.libre_menu_editor.LibreMenuEditor" was "needed", I assumed that all "-" would always be changed into "_" in the flatpak/flathub world.

If this was a mistake, I would be happy to correct it. Having all the names and links consistent is much nicer anyway.